var/spack/repos/builtin/packages/py-pymc3/package.py

```diff
@@ -17,6 +17,7 @@ class PyPymc3(PythonPackage):
 
     license("Apache-2.0")
 
+    version("3.11.6", sha256="9e930a1cfd2ee558892b4d92af043696c65a622b64098332687fd75c78f10bce")
```

Comment:
Confirmed the sha256 but the dependencies below don't match what this version needs per `requirements.txt`:

```
arviz>=0.11.0
cachetools>=4.2.1
deprecat
dill
fastprogress>=0.2.0
numpy>=1.15.0,<1.22.2
pandas>=0.24.0
patsy>=0.5.1
scipy>=1.7.3,<1.8.0
semver>=2.13.0
theano-pymc==1.1.2
typing-extensions>=3.7.4
```

The minimum python is also `3.7` per this version's `setup.py`.